cluster.tf

```diff
@@ -25,6 +27,22 @@ resource google_container_cluster management_cluster {
     identity_namespace = "${var.project_id}.svc.id.goog"
   }
 
+  datapath_provider = "ADVANCED_DATAPATH"
```

Comment:
@fristonio clearly this shouldn't need to be optional, right? ;)

Reply:
Yeah, this makes sense to me.